dispatch/apps/content/models.py

```diff
@@ -429,16 +437,26 @@ def get_medium_url(self):
         """
         Returns the medium size image URL.
         """
-        return "%s%s-%s.jpg" % (settings.MEDIA_URL, self.get_name(), 'medium')
+        fileName = re.split('.(jpg|gif|png)',self.get_name())
```

Comment:
- Change `fileName` to `file_name` (Python style convention)
- Add space between function arguments